keyboards/sofle_pico/keyboard.json

```
@@ -0,0 +1,251 @@
+            {"matrix": [1, 2], "x": 38, "y": 19, "flags": 4},
+            {"matrix": [0, 2], "x": 38, "y": 5, "flags": 4},
+            {"matrix": [0, 1], "x": 19, "y": 7, "flags": 4},
+            {"matrix": [1, 1], "x": 19, "y": 21, "flags": 4},
+            {"matrix": [2, 1], "x": 19, "y": 36, "flags": 4},
+            {"matrix": [3, 1], "x": 19, "y": 50, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 50, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 36, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 21, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 7, "flags": 4},
+            {"matrix": [5, 5], "x": 129, "y": 7, "flags": 4},
+            {"matrix": [6, 5], "x": 129, "y": 21, "flags": 4},
+            {"matrix": [7, 5], "x": 129, "y": 36, "flags": 4},
+            {"matrix": [8, 5], "x": 129, "y": 50, "flags": 4},
+            {"matrix": [9, 4], "x": 119, "y": 64, "flags": 4},
+            {"matrix": [9, 3], "x": 129, "y": 64, "flags": 4},
+            {"matrix": [9, 2], "x": 148, "y": 62, "flags": 4},
+            {"matrix": [8, 4], "x": 148, "y": 48, "flags": 4},
+            {"matrix": [7, 4], "x": 148, "y": 33, "flags": 4},
+            {"matrix": [6, 4], "x": 148, "y": 19, "flags": 4},
+            {"matrix": [5, 4], "x": 148, "y": 5, "flags": 4},
+            {"matrix": [5, 3], "x": 167, "y": 3, "flags": 4},
+            {"matrix": [6, 3], "x": 167, "y": 17, "flags": 4},
+            {"matrix": [7, 3], "x": 167, "y": 31, "flags": 4},
+            {"matrix": [8, 3], "x": 167, "y": 46, "flags": 4},
+            {"matrix": [9, 1], "x": 167, "y": 62, "flags": 4},
+            {"matrix": [9, 0], "x": 186, "y": 64, "flags": 4},
+            {"matrix": [8, 2], "x": 186, "y": 48, "flags": 4},
+            {"matrix": [7, 2], "x": 186, "y": 33, "flags": 4},
+            {"matrix": [6, 2], "x": 186, "y": 19, "flags": 4},
+            {"matrix": [5, 2], "x": 186, "y": 5, "flags": 4},
+            {"matrix": [5, 1], "x": 209, "y": 7, "flags": 4},
+            {"matrix": [6, 1], "x": 209, "y": 21, "flags": 4},
+            {"matrix": [7, 1], "x": 209, "y": 36, "flags": 4},
+            {"matrix": [8, 1], "x": 209, "y": 50, "flags": 4},
+            {"matrix": [8, 0], "x": 224, "y": 50, "flags": 4},
+            {"matrix": [7, 0], "x": 224, "y": 36, "flags": 4},
+            {"matrix": [6, 0], "x": 224, "y": 21, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 7, "flags": 4}
+        ],
+        "max_brightness": 127,
+        "split_count": [29, 29]
+    },
+    "split": {
+        "enabled": true,
+        "bootmagic": {
+            "matrix": [5, 0]
+        },
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP14", "pin_b": "GP15"}
+                ]
+            }
+        },
+        "serial": {
+            "driver": "vendor"
+        },
+        "transport": {
+            "sync": {
+                "layer_state": true
```

If you have rgb matrix enabled, and reactive animations enabled, then you'll want to enable the matrix state syncing too.
```suggestion
                "layer_state": true,
                "matrix_state": true
```